tests/components/glances/test_config_flow.py

```diff
@@ -54,7 +52,10 @@ async def test_form(hass):
 async def test_form_cannot_connect(hass):
     """Test to return error if we cannot connect."""
 
-    with patch("glances_api.Glances"):
```

Comment:
This was not correctly mocked out, we were actually trying to connect to the API, which failed.